memory.tex

```diff
@@ -1773,21 +1721,103 @@ \subsection{C64-style Memory Mapping}
 70 PRINT PEEK($D020)
 \end{screenoutput}
 
-Unlike \$D030 banking, C64-style banking occurs {\em after} address translation
-with the MAP register. If a program accesses a 16-bit address that is offset
-into the \$D000 -- \$DFFF region by the MAP register, and I/O register banking
-is enabled (the default), the program will access the I/O register.
+C64-style banking only applies to 8KB address blocks not selected by the MAP
+register (``unmapped'' blocks). For example, if C64-style banking has I/O
+registers at \$D000 -- \$DFFF, and the \$C000 -- \$DFFF region is mapped with
+an offset of 0, the program will access RAM at that location and {\em not} the
+I/O registers. The \$C000 -- \$DFFF region must be unmapped for the CPU to
+access the I/O registers.
+
+Similarly, it is possible to use the MAP register to access the RAM at
+addresses 0.0000 and 0.0001 that are normally hidden by the C64-style banking
+registers. If the MAP register selects the \$0000 -- \$1FFF address block for
+mapping and applies an offset of 0, 16-bit addresses \$0000 and \$0001 will
+access RAM at 0.0000 and 0.0001, and will not access the C64-style memory
```

Comment:
The term "C64-style banking registers" is used in this section, but here it is called "C64-style memory mapping registers". Should that be made consistent?

Reply:
Agreed, done. I'll go with "banking" because "mapping" is such an important concept that I don't want to confuse the two.